ddtrace/contrib/dbapi/__init__.py

```diff
@@ -154,6 +154,29 @@ def _set_post_execute_tags(self, span):
         if isinstance(row_count, six.integer_types) and row_count >= 0:
             span.set_tag(sql.ROWS, row_count)
 
+    def _propagate_dbm_context(self, dbspan, args):
+        # type: (...) -> Tuple[Tuple[Any, ...]]
+        dbm_comment = _database_monitoring._get_dbm_comment(dbspan)
+        if dbm_comment is None:
+            return args
+        # add DBM comment to query arg
+        sql_with_dbm_tags = self._dbm_sql_injector(dbm_comment, args[0])
```

Comment:
we can always rely on this order?

Reply:
Yes due to how we call TracedCursor._traced_method(…) in the execute/executemany/callproc. The sql statement is never passed as a keyword argument. It’s always the first argument. 

Relying on this behavior was the motivation for this refactor PR: https://github.com/DataDog/dd-trace-py/pull/4470.